src/services/MailService.js

```diff
@@ -3,17 +3,23 @@
 import {createTransport, SMTPTransport} from 'nodemailer'
 import {compileFile} from 'pug'
 import Offer from '../models/Offer'
+import cityConfigs from '../cities/cityConfigs'
+import moment from 'moment'
 
 const develop = process.env.NODE_ENV === 'development'
 
 const renderConfirmationMail = compileFile('src/views/confirmationMail.pug')
-const renderCreationMail = compileFile('src/views/creationMail.pug')
+const renderRequestConfirmationMail = compileFile('src/views/requestConfirmationMail.pug')
 const renderDeletionMail = compileFile('src/views/deletionMail.pug')
 const renderExtensionMail = compileFile('src/views/extensionMail.pug')
 
-const getConfirmationUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/confirm`
-const getDeletionUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/delete`
-const getExtensionUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/extend`
+const getConfirmationUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}/confirm`
+const getExtensionUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}/extend`
+const getDeletionUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}`
```

Comment:
These are not the correct urls. These lead to the API, whereas they should lead to the frontend.
The correct urls would be:
```js
const getConfirmationUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/confirm`
const getExtensionUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/extend`
const getDeletionUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/delete`
```

Reply:
Done